internal/receiver/smartagentreceiver/README.md

```diff
@@ -30,6 +37,7 @@ receivers:
     type: postgresql
     host: mypostgresinstance
     port: 5432
+    metadataClients: [signalfx]
```

Comment:
It's unfortunate that we need to use a different casing than what is used by Otelcol. This should be temporary. Once we settle on refactoring MetadataUpdates into Entity events this should be removed completely. If that happens then we will not need GetPipelines() either.

Reply:
That was my one of my thoughts (this is the y of the x/y problem).  It would be even be preferable to send meaningful datapoints w/ special labels and no metric value that can be converted to dimension updates if entities don't land.